### ISSUE TITLE
Closing the information box with Esc does not work after visiting different browser tab

### ISSUE THREAD

**Issue by Ansa211** [czcorpus/kontext#2536]
By "information box" I mean this one:

![moc_velka](https://user-images.githubusercontent.com/1833866/44633262-0daf3500-a989-11e8-8815-21e32815eb7c.png)

As can be seen in the image, when the window with KonText is small (or the content of the box very large), the "Close" button (a blue square with an X symbol) which normally appears in its right top corner becomes inaccessible. (On Lindat KonText, we use the same box for the output of a plugin; when the content is large, there is a slide bar, so one can at least reach the box by sliding all the way up.) Right after opening the box, it can also be closed by pressing the Esc button. However, if it becomes out of focus (e.g. when the user clicks a different window or a different tab of the browser), the Esc button no longer works for closing it (not even after clicking directly into the box). So there is no way to close the box in such cases other than either resizing the window with KonText or applying a different zoom (font size) to the webpage.

**Comment by Ansa211** [comment]
One more screenshot. The "Speeches" plugin is equipped with the slide bar, but even if one slides all the way up, the X button is hidden behind the black navigation bar:

![slide_to_x](https://user-images.githubusercontent.com/1833866/44669927-9bd6fa00-aa21-11e8-976f-e6e09a2315d8.png)

Maybe it would make sense if the X button would always float to the right top corner of the visible area instead of being fixed to the right top corner of the content of the plugin box.

**Comment by tomachalek** [comment]
I've made some tweaks (https://github.com/czcorpus/kontext/pull/2541) to make this problem less annoying: 

1. Now it is possible to regain focus for the box by clicking anywhere within its visible area.
1. The max-height property is set to a smaller value to prevent the box from occupying the top menu (or other similar stuff).

Speaking about fixed position of the "X button". I've already tried this but unfortunately, I wasn't able to do this properly - i.e. to have a fixed button AND keep all the other required properties of the box in terms of its styling (scrolling, positioning, sizing). So I keep this in mind but I wouldn't expect a complete solution anytime soon.
